S3/outputs.tf

```diff
@@ -0,0 +1,25 @@
+output "s3_bucket_id" {
+  description = "The name of the bucket."
```

Comment:
Same comment if we want `(Optional)` and `(Required)` suffixes in the var descriptions.